static/js/src/static-forms.js

```diff
@@ -157,7 +157,11 @@ function setUpStaticForms(form, formId) {
     submitButton.appendChild(spinnerIcon);
   }
   const submitButton = form.querySelector('button[type="submit"]');
-  if (submitButton) {
+
+  // Exclude forms that don't need loader
+  const cancelLoader = submitButton.classList.contains("no-loader");
+
+  if (submitButton && !cancelLoader) {
```

Comment:
Needed a way to cancel the loader after download, because unlike other forms we don't navigate to another page, so it keeps rolling the spinner forever.